taccsite_cms/static/site_cms/css/src/_imports/components/c-button.css

```diff
@@ -4,5 +4,5 @@
 .c-button {
   --max-width: unset;
 
-  font-size: var(--global-font-size--medium);
+  font-size: var(--global-font-size--small);
```

Comment:
**Explanation.** I had guessed the wrong font size when I first set up these untested CMS-specific `c-button` styles. Untested, even though CMS has buttons, because the first CMS button was not a `.c-button`, it was a Django CMS form button.[^1][^2]

[^1]: [`taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.forms.css`](https://github.com/TACC/Core-CMS/pull/560/files#diff-e9414234ff809b486ac95af5c2c1b4fd1cf2dd7618564c667529baa52133d99f)

[^2]: The first `c-button` came when "+ Add …" and "– Remove" buttons were added to [APCD "Request to Submit" form](https://apcd-qa.tacc.utexas.edu/register/request-to-submit/).